content/python/concepts/keywords/terms/class/class.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'class'
+Description: 'Used for defining classes in Python.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Classes'
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`class`** keyword is used for defining [classes](https://www.codecademy.com/resources/docs/python/classes) in Python.
+
+## Syntax
+
+```pseudo
+class ClassName:
+  # Class body
+```
+
+The name of the class should be in "CamelCase". The indented class body contains definitions for any attributes or methods. Class definitions cannot be empty, so the `pass` statement can be used as a placeholder to avoid errors.
```

Comment:
```suggestion
The name of the class should be in CamelCase. The indented class body contains definitions for any attributes or methods. Class definitions cannot be empty, so the `pass` statement can be used as a placeholder to avoid errors.
```

Reply:
Does this need to be in quotes

Reply:
Nope!